.github/dependabot.yml

```diff
@@ -54,3 +55,4 @@ updates:
         versions: ["6.x"]
       - dependency-name: "@storybook/theming"
         versions: ["6.x"]
+      - dependency-name: "jasmine-marbles"
```

Comment:
Wondering why we're ignoring this one?

Reply:
Sorry, forgot to explain this in the description. We have had the dependabot PR open for `jasmine-marbles` for a very long time. I have tried to update it multiple times but when running in the workflow it always fail (locally it's fine) and I can't figure out the reason.
If i don't ignore the dependency here it will try to add it to the combined PR each time and block it from being merged without manual intervention.

Reply:
Done 👍